### PR TITLE
Add documentation for DefaultActiveSpeakerPolicy constructor

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**
#675 

**Description of changes:**
- Add documentation for DefaultActiveSpeakerPolicy constructor

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes? Ran `npm run doc` and checked in chrome for the updated content
3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No
4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
+   * the updated score is returned as 0, else the updated score is returned.
+   *
+   * @param silenceThreshold
+   * Threshold number compared with current received volume.
+   * While calculating the new active speaker score, if the current received
+   * volume is less than this threshold value, the current received volume is considered as 0,
+   * else 1.
+   *
+   * @param takeoverRate
+   * Number used to calculate other attendees active speaker score, other than the current attendee.
```
Comment (simmkyu) [inline src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts:33]: `other attendees` => `other attendee's` or `other attendees'`

File: src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts [hunk 2]
```diff
@@ -1,11 +1,44 @@
...
 export default class DefaultActiveSpeakerPolicy implements ActiveSpeakerPolicy {
+  /**
+   * Map of attendeeIds to their active speaker score values
+   */
   private volumes: { [attendeeId: string]: number } = {};
 
+  /** Creates active speaker policy with speakerWeight, cutoffThreshold, silenceThreshold, and takeoverRate.
+   *
+   * @param speakerWeight
+   * Number used to calculate new active speaker score for current attendee
```
Comment (simmkyu) [inline src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts:15]: optional: start with an article in four fields, e.g. `Number` => `The number`  or `A number`

**Review by simmkyu — APPROVED** [review]

File: src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts (hunk 2 above)
Comment (devalevenkatesh) [reply src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts]: Changed for the fields, used `The`. Thanks!

File: src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts (hunk 1 above)
Comment (devalevenkatesh) [reply src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts]: Yes, corrected it thanks for catching.

**Comment by anuranduttaroy** [comment]
This issue https://github.com/aws/amazon-chime-sdk-js/issues/630 seems to be related to this PR as well.

**Review by simmkyu — APPROVED** [review]

**Comment by devalevenkatesh** [comment]
> This issue #630 seems to be related to this PR as well.

Yes, I think we can close both since @simmkyu already has addressed #630 . If there is some performance problem with the `DefaultActiveSpeakerPolicy` itself, separate issue can be tracked. Thanks for mentioning it here @anuranduttaroy.

**Review by anuranduttaroy — APPROVED** [review]

**Pull request merged**